src/main/java/net/sf/jabref/bibtex/BibEntryWriter.java

```diff
@@ -108,11 +108,11 @@ private void writeRequiredFieldsFirstRemainingFieldsSecond(BibEntry entry, Write
             }
         }
         for (String field : remainingFields) {
-            writeField(entry, out, field, indentation);
+            hasWritten = hasWritten | writeField(entry, out, field, hasWritten, indentation);
```

Comment:
What is going on in this class? Why and what did you change here? Didn't someone recently remove the hasWritten stuff? @tobiasdiez ?